Tests/SonarScanner.MSBuild.PreProcessor.Test/Infrastructure/MockSonarWebService.cs

```diff
@@ -129,7 +128,7 @@ Task<bool> ISonarWebService.TryDownloadEmbeddedFile(string pluginKey, string emb
             }
         }
 
-        public Task<AnalysisCacheMsg> DownloadCache(string projectKey, string branch) =>
+        public Task<IList<SensorCacheEntry>> DownloadCache(string projectKey, string branch) =>
             Task.FromResult(projectKey == "key-no-cache" ? null : Cache);
```

Comment:
As the real implementation returns an empty list instead of null, shouldn`t the mock have the same behavior?